README.md

```diff
@@ -2,6 +2,8 @@
 
 🤖 A GitHub Action that provides AI-powered code reviews for your pull requests using multiple LLM providers. Created by a very smart otter with multiple degrees in Computer Science! 🎓
 
```

Comment:
🖼️ Great addition of the OtterAI image! Ensure the image path is correct and the image is optimized for web use.